### DISCUSSION TITLE
Welcome to S3E Discussions!

### DISCUSSION THREAD

**Discussion by DapengFeng** [DapengFeng/S3E#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by whu-lyh** [comment]
how to download your dataset?

**Reply by DapengFeng** [reply]
the dataset is available on https://huggingface.co/datasets/PengYu-Team/S3E/tree/main .

**Reply by whu-lyh** [reply]
Many thanks, I want to know how to parse the db3 files? I am new here

**Reply by DapengFeng** [reply]
Thank you for your comment, please refer to #21 . We will update our readme and project website nearly.

**Comment by zyd-123** [comment]
Can you provide the dataset for ros1?

**Reply by DapengFeng** [reply]
Since we record the messages with the standard type, converting from ROS 2 to ROS 1 using the [`rosbags-convert`](https://ternaris.gitlab.io/rosbags/topics/convert.html) tool is straightforward.

**Comment by 8613425400769** [comment]
请问在DCL-SLAM当中的run.launch中，<arg name="SYSU_bag" value="$(env HOME)/rosbag-data/S3E/SYSU_LIBRARY.bag"/>                   <arg name="bag_file3" value="$(env HOME)/2022_07_22_19_48_01_rect.bag"/>
	<arg name="bag_file4" value="$(env HOME)/2022_07_22_19_54_37_rect.bag"/>
	<arg name="bag_file5" value="$(env HOME)/2022_07_22_20_01_00_rect.bag"/>这些数据集对应的是S3E当中哪些数据集？
